src/seedsigner/gui/components.py

```diff
@@ -568,6 +570,52 @@ def render(self):
             self.icon.render()
 
 
+@dataclass
+class ToastOverlay(BaseComponent):
+    icon_name: str = None
+    color: str = None
+    label_text: str = None
+
+    def __post_init__(self):
+        super().__post_init__()
+            
+        self.icon = Icon(
+            image_draw=self.image_draw,
+            canvas=self.canvas,
+            screen_x=20,
+            screen_y=190,
+            icon_name=self.icon_name,
+            icon_size=30,
+            icon_color=self.color
+        )
+        
+        self.label = TextArea(
+            image_draw=self.image_draw,
+            canvas=self.canvas,
+            text=self.label_text,
+            font_size=19,
+            font_color=self.color,
+            edge_padding=0,
+            is_text_centered=False,
+            auto_line_break=False,
+            width=160,
+            height=20,
+            screen_x=55,
+            screen_y=195,
+            allow_text_overflow=False
+        )
+            
+    def render(self):
+        self.image_draw.rounded_rectangle(
+            ( 10, 180, 230, 230),
```

Comment:
Whenever possible, use `GUIConstants` spacers or multiples on the spacers. e.g.:

```
GUIConstants.EDGE_PADDING * 2
```

Arguably the for `radius` and `width` below, though I think those sorts of pixel values are less consistently handled elsewhere.

The code isn't really totally flexible for any future display size, but whatever efforts we can make to partially pre-bake that in will help.